app/models/manageiq/providers/google/cloud_manager/provision/cloning.rb

```diff
@@ -24,6 +24,11 @@ def prepare_for_clone_task
     # issue: https://github.com/fog/fog-google/issues/136
     clone_options[:on_host_maintenance] = "TERMINATE" if clone_options[:preemptible]
 
+    unless clone_options[:user_data].nil?
```

Comment:
Prefer the positive case `if clone_options[:user_data]`

Reply:
👍 Me too. Thanks!